src/api/api.ts

```diff
@@ -77,22 +78,22 @@ export const getMember = async (client: IDeskproClient, email: string): Promise<
       return null;
     }
 
-    if (data.exact_matches.members.length !== 1) {
-      return null;
-    }
-
-    const member = data.exact_matches.members.pop();
-
-    return {
-      id: member.id,
-      webId: member.web_id,
-      email: member.email_address,
-      fullName: member.full_name,
-      rating: member.member_rating,
-      status: member.status,
-    };
+    return (data.exact_matches.members ?? []).map((m: any) => ({
```

Comment:
Its not clear to me why you need to type `m` as `any`. Surely this can be infered from the type of `data.exact_matches.members`?

Reply:
It might be better to at least just type the expected result of `await response.json()`, and maybe consider adding some validation to the response. I can recommend `zod` as a library for validating data.
```ts
const data: SomeReponseType = await response.json();

```